### DISCUSSION TITLE
Mobile documentation view/edit

### DISCUSSION THREAD

**Discussion by mkbehbehani** [community/community#20825] [Mobile]
GitHub product folks: we’d like to migrate off of Atlassian but GitHub needs to check off a last item: support for mobile read/write of documentation. 

- It’s a tiny feature but it prevents us from being able to say “GitHub does everything in our workflow, we now have equivalent project features to Jira and equivalent documentation features to confluence”. 

- Typical use case: Right now if a slack question is sent while a dev is out and about, it’s low friction for them to find and  share related instructions in Confluence’s mobile app. If we were to migrate to GitHub it might be “I’ll find the doc in the repo wiki and send it to you when I’m back at my computer” and a few hours or next-day before that follow-up answer with the link appears.

**Comment by iskae** [comment]
Hi @mkbehbehani 👋 

Thanks for reaching and your first ever feature request in GitHub Mobile 🎉 We would love to support you and make your workflow as easy as possible 😊 

I would love to understand your request a little bit more. Assuming that your documentation are simple markdown files you should be able to share and render them on our native mobile apps. Is it not the case? 😅 

Based on the previous assumption creating documentation/ editing markdown files are not supported currently, but we also have some plans for it, so please stay tuned for our future releases 😊 

Please let me know if I understood your request totally wrong 🙇

**Reply by mkbehbehani** [reply]
Hi @iskae thank you for getting back to me. 

Put another way:
Context - GitHub has a wiki feature for the repo. Our team members like working with the GitHub wiki via desktop browser. We want to move docs from confluence into this feature. 

![Screen Shot 2022-07-23 at 11 15 59 AM](https://user-images.githubusercontent.com/1660550/180613538-9b43fc4c-960b-4754-99ac-92c52f81522d.png)

Scenario 1: An engineer is out running errands and gets a slack message from a junior dev asking how to set up something. There is an instruction page in the repo wiki. How can the member open and share that wiki page from GitHub's mobile app?

Scenario 2: The team member notices a missing step in the instructions and wants to quickly add that step. How can the member make that update from the mobile app?

These are the current workarounds I am aware of:
- Exit the mobile app, use the mobile browser - Not a good experience
- Don't use the wiki, move the docs into markdown files in the repo - Higher friction, edits likely require pull requests.
- Wait until back at desktop.

Short term suggestion: Since the wiki today is a git repo under `reponame.wiki`, if I could access and edit that repo using the existing mobile features, that might be good enough. AFAIK I just can't access at that repo from the mobile app at the moment.
